### PR TITLE
Remove references to accman/pull_keys.

### PR DESCRIPTION
**Pull request by grugnog**

### CODE REVIEW

File: docs/03-policies/security.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Developers and themers maintaining local sandbox copies of client sites must als
 Developers and themers working on the site vhost (SSH/shell, file system, database) must also:
 
 * Ensure they follow best practices with respect to SSH keys, passphrases and passphrase caching (see above).
-* SSH, SFTP and SCP access to vhosts should only ever be through our central pull keys SSH key distribution system. Access by password, manually installed SSH key, web based "shell" script, port forwarding to 3rd parties or other methods are forbidden by unless authorized in advance by the CIO.
+* SSH, SFTP and SCP access to vhosts should only ever be managed by designated admin(s). Access by password, manually installed SSH keys (other than by admins), web based "shell" script, port forwarding to 3rd parties or other methods are forbidden by unless authorized in advance by the CIO.
```
Comment (openprivacy) [inline docs/03-policies/security.md]: For clarity and conciseness: replace "_should only ever be managed_" with "_are managed exclusively by_"

**Review by openprivacy — CHANGES_REQUESTED** [review]
Thanks for this update! See comment, and please bump the version number at the top to `1.1.2`

**Review by openprivacy — APPROVED** [review]

**Review by openprivacy — APPROVED** [review]
Looks great - thanks!

**Comment by kevwalsh** [comment]
@openprivacy looks like you approved but forgot to merge? or are there further changes needed?

**Comment by openprivacy** [comment]
@kevwalsh I reviewed and accepted, but no place to merge? (I'd merge if I could...) Page says"Code owner review required" - an I missing something?

**Comment by kevwalsh** [comment]
@openprivacy i'll merge this, and i'll look into why you couldn't. i guess cause you're not a member of the docs or management team, and one of the files touched is /docs/03-policies/security.md ... we should make you a codeowner of that file in particular.  Changes to anything in /docs/03-policies must be approved by someone in mgmt.

**Review by kevwalsh — APPROVED** [review]

**Pull request merged**